### PR TITLE
Port identity function to rust

### PR DESCRIPTION
**Pull request by kidd**

### CODE REVIEW

File: rust_src/src/lib.rs [hunk 1]
```diff
@@ -64,6 +64,7 @@ mod editfns;
 mod util;
 mod minibuf;
 mod cmds;
+mod fns;
```
Comment (birkenfeld) [inline rust_src/src/lib.rs]: I'd put it in `objects.rs` instead.

Comment (kidd) [reply rust_src/src/lib.rs]: I wanted to keep the file structure the same as with original emacs sources, but I have no problem moving it to objects.rs . will do it right away. 

Thanks!

Comment (birkenfeld) [reply rust_src/src/lib.rs]: We've sort of picked apart fns.c already since it is such a grab-bag. Thanks for the update!

**Comment by kidd** [comment]
ready to be merged :)

File: rust_src/src/objects.rs [hunk 2]
```diff
@@ -52,3 +52,9 @@ fn equal_including_properties(o1: LispObject, o2: LispObject) -> LispObject {
     };
     LispObject::from_bool(res)
 }
+
+/// Return the argument unchanged.
+#[lisp_fn]
+fn identity(x: LispObject) -> LispObject {
```
Comment (birkenfeld) [inline rust_src/src/objects.rs]: argument should be called `arg` like in the C version.

**Comment by Wilfred** [comment]
@kidd I've added you as a collaborator! This enables you to review other people's PRs too, if you wish :)

**Pull request merged**